store/store.go

```diff
@@ -173,6 +176,16 @@ type Store struct {
 	suggestedCurrency string
 }
 
+func httpStatusErrToError(httpErr httpErrorResponse, msg string) error {
+	tpl := "cannot %s: got unexpected HTTP status code %d via %s to %q"
+	if httpErr.Oops != "" {
+		tpl += " [%s]"
+		return fmt.Errorf(tpl, msg, httpErr.StatusCode, httpErr.Method, httpErr.URL, httpErr.Oops)
+	}
+
```

Comment:
Something is wrong here. Either you only want to return an error when there is an oops, in which case whole tpl + adding to it is nonsense, or you forgot half of the function. If it's the latter it's alarming that tests have passed